src/openlrr/game/interface/TextMessages.cpp

```diff
@@ -1,7 +1,17 @@
 // TextMessages.cpp : 
 //
 
+#include "string.h"
```

Comment:
This include should be in angle brackets, because it's not part of OpenLRR's codebase.

However `common.h` (which is included by `TextMessages.h` or headers it references) already includes `<cstring>`, so that should contain everything that `<string.h>` has already. Generally any include you need may already be in `common.h`.